libs/data-mapper/src/lib/models/Schema.ts

```diff
@@ -53,6 +53,11 @@ export enum NormalizedDataType {
   String = 'String',
 }
 
+export enum InputFormat {
+  TextBox = 'TextBox',
+  FilePicker = 'FilePicker',
```

Comment:
I assume this does nothing for now?

Reply:
Yeah so this will work as soon as backend pushes these changes, but not yet.